### ISSUE TITLE
Don't include FIXED bugs for nightly

### ISSUE THREAD

**Issue by overholt** [mozilla/releasehealth#18]
It'd be nice if we didn't include FIXED bugs in the counts (or queries) for nightly since once something is fixed there we don't need to uplift anywhere. I could write a patch but it'd end up with some ugly if {...} blocks.

**Comment by lmandel** [comment]
Yes. Good catch. We should just use resolution for Nightly. This will require changing the queries as you noted. Will have to think about how best to do that. Short term we can remove fixed items from the queries by changing the status flags to fixed.

**Comment by pascalchevrel** [comment]
Currently the only fixed bugs we include in search results (across channels) are duplicates. I don't think it makes sense for a dashboard to show duplicates as active bugs, the only case when I am interested in looking at duplicates is to double check if the fix worked for the original bug (was is really a duplicate) but I don't think it makes sense to have both the duplicates and the bug where the work will happen be cumulated in the search numbers.
